supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -4,64 +4,75 @@
 
 The following guidelines improve findability and search engine optimization.
 
-.Benefits of internal linking
```

Comment:
"Internal linking" to me might mean links within an intranet, or within a single page...maybe that's just me?
w3 uses the term "bookmarks"[1], but honestly, I don't know if that's any better...curious to know if "internal linking" is intuitive to most people.

[1] https://www.w3schools.com/html/html_links_bookmarks.asp

Reply:
I agree @stoobie it's not crystal clear really. Internal to me means within the same guide most of the time, but that's not what this original content meant I think.

Reply:
I've seen some discussions on what "internal links" mean. From an SEO/ Findability perspective internal links can mean a few things.

1. a link in a document to another part of the same document
2. a link in a document to another part of the same property (access.redhat.com/page1) to (access.redhat.com/page2)
3. a link in a document to another part of the TLD "top level domain" (access.redhat.com/page1) to (redhat.com/page1)
4. a link in a document to another property in the network of sites (access.redhat.com/page1) to (developers.redhat.com/page1)

All four of those examples are considered an "internal link" by Google - even if it's owned & maintained by the same org.
an external link points to a page outside the TLD (access.redhat.com/page1) to (opensource.com/page1) or (access.redhat.com/page1) to (whitehouse.gov/ostp) - Office of Science & Technology Policy

It's fair to have other definitions of what an internal or external link, I'm not trying to argue that this is the only definition, but for the purposes of what Google considers to be internal v external, this is how it works.

Reply:
@ndeevy I propose that at the beginning of this section, you define what an "internal" and "external" link is:
internal: Link within redhat.com.
external: Link to a domain other than redhat.com.
For links within a single document, I suggest stating that clearly "links within a single document".